docs/architecture/self-hosted/upgrade-backup-restore.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Upgrades and backups
+id: upgrade-backup-restore
+description: Learn how to upgrade, back up, and restore Prophecy.
+sidebar_position: 4
+tags:
+  - upgrade
+  - backup
+  - restore
+  - self-managed
+---
+
+This page outlines different actions you may perform to maintain your Prophecy installation.
+
+## Upgrade
+
+To upgrade Prophecy, you can do so in **Settings > Admin**.
+
+1. Navigate to the **Admin** tab of the Prophecy **Settings** page.
+2. Click on **Upgrade Version**.
+3. Choose the version you would like to upgrade to in the **Version** dropdown.
+4. Make sure that **Disable Rollback** toggle is on.
+
+![Upgrade version in Admin settings](img/upgrade-version.png)
+
+:::note
```

Comment:
Could we move the note up before the steps?

Reply:
yes!